docs/source/developing/writing-docs.adoc

```diff
@@ -12,85 +12,149 @@ toc::[]
+
+Minishift documentation is located in the *_docs_* sub-folder. The
 documentation is a mix of generated
 https://en.wikipedia.org/wiki/Markdown[Markdown] files and manually
 maintained https://en.wikipedia.org/wiki/AsciiDoc[Asciidoc] files.
 
-If you are adding a new file to the doumentation please update
- the `_topic_map.yml`  in the root of `docs/source` sub-folder.
-
-Per default the documentation is built in a Docker container. This way
+By default, the documentation is built in a Docker container. This way
 you avoid having to install all the required dependencies on your
 development machine. All you need is a running Docker daemon. In case
-you don't have one, use Minishift itself. For further information refer to
-link:../command-ref/minishift_docker-env{outfilesuffix}[`minishift docker-env`].
+you don't have one, use Minishift itself. For more information, see the
+link:../command-ref/minishift_docker-env{outfilesuffix}[`minishift docker-env`] command.
 
 To build the Docker image, run:
 
-[source,sh]
 ----
 $ make build_docs_container
 ----
 
-To generate the documentation into the directory `docs/build`, run:
+To generate the documentation into the directory *_docs/build_*, run:
 
-[source,sh]
 ----
 $ make gen_docs
 ----
 
 To build and serve the documentation for editing, run:
 
-[source,sh]
 ----
 $ make serve_docs
 ----
 
-The latter will start the https://middlemanapp.com[Middleman] server on
+The `make serve_docs` command starts the link:https://middlemanapp.com[Middleman] server on
 port 4567. You can access the rendered documentation by browsing to
-http://<IP of Docker daemon>:4567.
+_http://<IP-of-Docker-daemon>:4567_.
+
+If you encounter issues with the local staging of the documentation, you can run the following
+command to delete the *_build_* sub-folder and the variables file:
 
-[[section-deploying-minishift-docs]]
-== Deploying Minishift documentation
+----
+$ make clean_docs
+----
+
+[[deploying-docs]]
+== Deploying the documentation
 
 The Minishift documentation is deployed on _docs.openshift.org_ under
-link:https://docs.openshift.org/minishift[https://docs.openshift.org/minishift].
+link:https://docs.openshift.org/latest/minishift[https://docs.openshift.org/latest/minishift].
 
 To integrate with the _docs.openshift.org_ we deliver a tarball containing
-the Minishift asciidoc files as well as some link:http://www.asciibinder.org/[AsciiBinder]
+the Minishift AsciiDoc files as well as some link:http://www.asciibinder.org/[AsciiBinder]
 meta data.
 
 This tarball can be built using:
 
-[source,sh]
 ----
 $ make gen_adoc_tar
 ----
 
-Once the build completes there will be a _minishift-adoc.tar_ in the _docs/build_ directory
-of your checkout.
-
-How this tarball is integrated into the docs.openshift.org site build is described in
-<<section-building-openshift-docs>>.
+After the build completes there will be a *_minishift-adoc.tar_* file in the *_docs/build_* directory
+of your local repository.
 
```

Comment:
I don't think this is necessary, but now that I look at the section again it does seem to be missing a "why", as in what is the use-case for building the openshift docs (when would you want to do it). I'll investigate.